sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Customized/Models/TimeSeriesIdProperty.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// A definition of a single property that can be used in time series ID properties defined during environment creation.
+    /// </summary>
+    [CodeGenModel("TimeSeriesIdProperty")]
+    public partial class TimeSeriesIdProperty
+    {
+        /// <summary> Initializes a new instance of TimeSeriesIdProperty. </summary>
```

Comment:
minor: extra spaces inside the tags

Reply:
If you meant before "Initializes" and after "TimeSeriesIdProperty.", that is followed in other classes as well as in ADT.